src/Notes/WC_Admin_Notes_Filters.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * WooCommerce Notes Filters
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+ namespace Automattic\WooCommerce\Admin\Notes;
+
+ defined( 'ABSPATH' ) || exit;
+
+ /**
+  * This class adds actions to manage notes.
+  */
+class WC_Admin_Notes_Filters {
+	/**
+	 * Init
+	 */
+	public function init() {
+		add_action( 'activate_jetpack/jetpack.php', array( $this, 'action_jetpack_woocommerce_services_activation_note' ) );
```

Comment:
Nice! 💯 

Do you think we should move these to a constructor in `WC_Admin_Notes_Install_Jetpack_And_WooCommerce_Services_Plugin` and initialize that somewhere to keep the logic contained?  My concern is that this class could become a dumping ground for all note filters.

Reply:
I think that's a premature refactor, I like how there's separation between the hook and `WC_Admin_Notes_Install_Jetpack_And_WooCommerce_Services_Plugin` so I would rather keep that for now at least. Save refactoring it until this does actually start to become a dumping ground and the way forward for refactoring it becomes clearer.

Reply:
Ok, sorry, I just don't know the system well enough yet. Will do.

Reply:
Fixed in 0aa9c0386d318abab2432fbf4c6a073be27e3cd3